config/data/langs.toml

```diff
@@ -182,21 +182,19 @@ Next
 args       = [ '/usr/bin/befunge', '-' ]
 experiment = 428
 size       = '2.81 MiB'
-version    = 'Befunge-98 1,001'
+version    = 'b7bf640'
 website    = 'https://github.com/VorpalBlade/cfunge'
 example    = '''
 v printing
 > "!dlroW ,olleH" ,,,,,,,,,,,,, v
-v                           ,+19<
+v                             ,a<
```

Comment:
So apparently `a` is `10`, and me trying to figure out what she needed to do `19` and `91` times.

Reply:
in fact - now that we've swapped to befunge-98 that gives us access to `k` which pops the top number off the stack and repeats the following command n times, so we can replace the 13 commas with `ck,` (c being 13) 
```
v printing
> "!dlroW ,olleH" ck, v
v                   ,a<
v looping
> 0 v            <
    >:.a, 1+ :9`!|
v               $<
v accessing arguments
> #@~ :0`!#v_ ,
^        ,a<
```

Reply:
> in fact - now that we've swapped to befunge-98 that gives us access to `k` which pops the top number off the stack and repeats the following command n times, so we can replace the 13 commas with `ck,` (c being 13)

Done!